src/core/ucc_global_opts.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCC_GLOBAL_OPTS_H_
+#define UCC_GLOBAL_OPTS_H_
+
+#include "config.h"
+
+#include "utils/ucc_parser.h"
+#include "utils/ucc_log.h"
+
+typedef struct ucc_global_config {
+    /* Log level above which log messages will be printed*/
+    ucc_log_component_config_t log_component;
+
+    /* Coll component libraries path */
+    char *component_path;
+    char *component_path_default;
+    int   initialized;
```

Comment:
According to ucx codestyle should be aligned by *

Reply:
@Sergei-Lebedev but this is how our clang-format does this. I think we agreed that we are not "ucx codestyle". We have our own style: clang-format + some tweaks. Do we want to fix that manually as well? I'm fine with "*" not used for alignment. @manjugv @alex--m ?

Reply:
Fine by me as well, just wanted to make sure that we don't follow ucx codestyle in this case

Reply:
Fine by me.